lib/imagemounter/utils/list_developer_image.js

```diff
@@ -0,0 +1,182 @@
+import _ from 'lodash';
+import { zip, net } from '@appium/support';
+import { homedir } from 'os';
+import { join as joinPath } from 'path';
+import fetch from 'node-fetch';
```

Comment:
please use axios instead

Reply:
Is there any hidden issue to use `fetch`? I haven't use axios before, I could take some time learning it

Reply:
I just don't see a need in mixing different modules. axios is already used as the `standard` approach in other modules